tfjs-backend-webgl/src/mulmat_packed_gpu.ts

```diff
@@ -83,7 +85,7 @@ export class MatMulPackedProgram implements GPGPUProgram {
 
     this.userCode = `
       ${activationSnippet}
-
+      // Don't use uniform for sharedDimensionPacked for performance.
       const float sharedDimension = ${sharedDimensionPacked}.0;
```

Comment:
Note: Change `sharedDimensionPacked` to uniform will greatly impact the performance. Maybe it's because the uniform will prevent loop unrolling, and then some driver optimization path couldn't be reached.